Add pass_filenames: false to examples

bplotnick opened the pull request:
When specifying the paths to run for `opa-test`, it isn't sufficient to just add them to `args` since pre-commit will append the changed files to the command line, you also need to add `pass_filenames: false`

I'm just guessing that conftest has the same issue, but i have not tested it

anderseknert commented:
Thanks Ben! I think it would be good to add this as default in `.pre-commit-hooks.yaml`. WDYT?

bplotnick commented:
Yeah I think that makes sense. I've updated this to remove the argument from the REAMDE and add it to the hook.

I've also removed it from the readme for conftest. Reading about it a little bit more, it seems like conftest takes the files that changed for validation. The limitation is that if you change the policies, we want to probably run conftest on everything, but i think the default of just running on the changed config files is fine. People should be running `pre-commit run --all-files` in CI anyway.

To handle both cases, I think you'd need something a bit more sophisticated that takes into account whether the policies have changed or whether the config has changed. Maybe some shell script that does `find`.

anderseknert commented:
Makes sense. I'll make a note to check how this works with conftest and will push an update if needed. Thanks for the heads up!

anderseknert commented:
Just pushed a new version @bplotnick - should hopefully be detected when running `pre-commit autoupdate`. Thanks again for the fix 👍
